core/web/resolver/auth.go

```diff
@@ -3,16 +3,52 @@ package resolver
 import (
 	"context"
 
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/sessions"
 	"github.com/smartcontractkit/chainlink/core/web/auth"
 )
 
-// Authenticates the user from the session cookie.
+// Authenticates the user from the session cookie, presence of user inherently provides 'view' access.
 func authenticateUser(ctx context.Context) error {
-	_, ok := auth.GetGQLAuthenticatedSession(ctx)
+	if _, ok := auth.GetGQLAuthenticatedSession(ctx); !ok {
+		return unauthorizedError{}
+	}
+	return nil
+}
+
+// Authenticates the user from the session cookie and asserts at least 'run' role.
+func authenticateUserCanEditMinimal(ctx context.Context) error {
```

Comment:
This says "editMinimal". Ought it to say "run" instead? Perhaps worth grepping for other missed renames.

Reply:
Ah dang, good catch thanks. Updated the test harness code as well